### ISSUE TITLE
Change of pom group and artifactid?

### ISSUE THREAD

**Issue by lausdahl** [INTO-CPS-Association/FMI2-VDM-Model#14]
Would it be possible to set a "real" pom route and artifactid for this project. I would suggest to add the following:

This enabled os to deploy through sonatype
```
<!-- Needed to make deployment work -->
    <parent>
        <groupId>org.sonatype.oss</groupId>
        <artifactId>oss-parent</artifactId>
        <version>7</version>
    </parent>
```

and then we need to change the artefact and group something under a domain we own. So a into-cps.org could work or any sub domain under that or just an artefact id that isn't taken already.
```
    <groupId>org.into-cps</groupId>
    <artifactId>???</artifactId>
```

I'm asking for this because we would like to use this project as a maven dependency, and pushing it to maven central would allow any one to depend on it without any additional configuration.

**Comment by nickbattle** [comment]
Makes sense to me. So `org.into-cps` for the group and (say) `vdmcheck` for the artifactId?

**Comment by lausdahl** [comment]
That would make sense yes. We are already in control of this in sonatype so we can give raise a ticket to make an account to you for the release prodecure. Works fine with maven release. Then after is deployed there is a site where they run checks and allow you to more it from staging to production if the checks pass. The it will replicate to all maven central servers over c. 24 hours after that.

**Comment by nickbattle** [comment]
These POM changes have been made. Can we close this issue?

**Comment by CThuleHansen** [comment]
Ya. I guess a good follow-up would be deploy to sonatype? :D

**Comment by lausdahl** [comment]
fixed in ab8e3b4
